x/rollup/keeper/keeper.go

```diff
@@ -27,3 +30,17 @@ func NewKeeper(
 		rollupCfg:    &rollup.Config{},
 	}
 }
+
+// Helper. Prepares a `message` event with the module name and emits it
+// along with the provided events.
+func (k *Keeper) EmitEvents(goCtx context.Context, events sdk.Events) {
```

Comment:
It may be worth naming `EmitEvents` to something like `EmitMessageEvents`, `EmitRollupEvents`, or `EmitModuleEvents` to avoid confusion with the event manager `EmitEvents` func.

Reply:
There are a few spots in the codebase where overlapping function names are included, the idea being that the package / struct that they are attached to effectively namespaces them.

I think I'll leave this as-is for now, but happy to revise in future if it provokes any confusion.